### ISSUE TITLE
skip detection of non amd gpus

### ISSUE THREAD

**Issue by logicminds** [Ricks-Lab/amdgpu-utils#51]
I am running the amdgpu driver but the i915 GPU is getting in the way.  Please skip this onboard gpu if detected. 

```
Using python 3.6.9
           Python version OK.
Using Linux Kernel 4.20.0-042000-generic
           OS kernel OK.
AMD GPU driver is driver=i915 latency=0 
           AMD's 'amdgpu' driver package is required.
```

**Comment by Ricks-Lab** [comment]
@logicminds   Sorry for the delay in addressing this.  I must have missed the notification.  I made a change to amdgpu-chk.  Was that the only place where the error occurred?  Are you able to run the other utilities?

**Comment by Ricks-Lab** [comment]
@logicminds I have completely rewritten the way GPUs are read in by the utilities, and have eliminated the use of lshw. This should address the issue. It’s in the v3.0 branch which is now available for testing.

**Comment by Ricks-Lab** [comment]
Fixed in v3.0 which is now merged into master.
